framework/src/modules/network/network.js

```diff
@@ -290,7 +306,8 @@ module.exports = class Network {
 
 				return getCountByFilter(peerList, action.params);
 			},
-			applyPenalty: action => this.p2p.applyPenalty(action.params),
+			applyPenalty: action =>
+				this.p2p.applyPenalty(action.params.peerId, action.params.penalty),
```

Comment:
We should make the param name consistent with the one in `requestFromPeer` and `emitToPeer` actions.
`requestFromPeer` and `emitToPeer` use `action.params.peer` but `applyPenalty` uses `action.params.peerId`.

Maybe we should rename all to `action.params.peerId`?